setup-environment-internal

```diff
@@ -407,6 +407,23 @@ Your build environment has been configured with:
 
 You can now run 'bitbake <target>'
 
+EOF
+
+if [ -z "$LMP_VERSION_CACHE" ]; then
+	cat <<EOF
+LMP state cache is disabled
+
+EOF
+else
+	cat <<EOF
+LMP state cache is enabled:
+    LMP_VERSION_CACHE = ${LMP_VERSION_CACHE}
+    SSTATE remote url = https://storage.googleapis.com/lmp-cache/v$LMP_VERSION_CACHE-sstate-cache
+
+EOF
+fi
+
+cat <<EOF
```

Comment:
The LMP state cache is enabled by default if no other is defined

https://github.com/foundriesio/lmp-manifest/blob/01323b77f66f14891586351c9aab73963218810c/setup-environment-internal#L263-L264


but  because the value can be different depending where it runs (ci/local) maybe the best option is asking bitbake:
```
echo "LMP state cache mirror is: $(bitbake-getvar SSTATE_MIRRORS | tail -n1)"
```
This also will show when it is disabled.

Reply:
@quaresmajose